### ISSUE TITLE
Enhancement request: custom order of control elements

### ISSUE THREAD

**Issue by damyanor** [nervetattoo/simple-thermostat#126]
I would like to have an option to set the order of hvac/preset/fan/swing elements.

**Comment by FredericMa** [comment]
Same here, unless we are not talking about the same thing.
I defined my presets like this:
```yaml
    control:
      preset:
        eco:
          name: Uit
          icon: hass:power
        comfort:
          name: Verwarmen
          icon: hass:fire
```
Buth they are shown the other way around (alphabetically?):
![image](https://user-images.githubusercontent.com/7840263/83641852-61af6a80-a5ae-11ea-9fb2-4a09f89ac731.png)

**Comment by damyanor** [comment]
I've got exactly the same thing on my mind.

**Comment by nathbullen** [comment]
On top of that, I've got a second row of control for fan speed, but it organises it alphabetically after every HA reboot, and places the fan control buttons above the HVAC control buttons

**Comment by reversebias** [comment]
Lovelace problem. Fixed in 0.115.

**Comment by FredericMa** [comment]
I've upgraded to 0.115.3 but in my case the presets are still in reverse order shown.
So comfort and eco while I define them in yaml as eco and comfort:
> Same here, unless we are not talking about the same thing.
> I defined my presets like this:
> 
> ```yaml
>     control:
>       preset:
>         eco:
>           name: Uit
>           icon: hass:power
>         comfort:
>           name: Verwarmen
>           icon: hass:fire
> ```
> 
> But they are shown the other way around (alphabetically?):
> ![image](https://user-images.githubusercontent.com/7840263/83641852-61af6a80-a5ae-11ea-9fb2-4a09f89ac731.png)
